data/donations.json

```diff
@@ -222,5 +222,277 @@
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "c538c922-98d9-4b93-8589-7e8aaea82b83",
+        "firstName": "Anonymous",
+        "sum": 25,
+        "userId": "24093c95-df35-45f4-8c38-14f0cf6f2134",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "34fe3b48-addd-4b2c-b388-1a8f6be7a714",
+        "sum": 25,
+        "userId": "51ce0c32-1327-4e7f-b18e-867997271eeb",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "55edef29-3f05-4e32-96cc-7374bbd56004",
+        "sum": 25,
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "dcf68fe2-f3cc-437d-a442-d10fd587db2a",
+        "firstName": "Vladimir",
+        "sum": 25,
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "bc1e46c4-56c8-4f45-8dbc-d08827335ce3",
+        "firstName": "Anonymous",
+        "sum": 25,
+        "userId": "f1036d1b-4f0b-4f4c-b963-62aa2cc5ca80",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "a1549382-f6b3-4149-aebd-c4d44407b333",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "758f2314-4680-4853-a77f-69072a9ddad4",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "0a5fa3cf-7bb1-4e14-bbad-227e69d600d4",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "32d8b57f-f698-4cc3-a0da-78f63b28ada9",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "userId": "c7ce4318-6dd8-4634-b613-99f44b0e95c0",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "d3df3613-b54b-4de0-bc18-98b7787f947d",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "userId": "ff0f0dc7-8cb1-4e7b-aa64-68beeddafb75",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "2ae9a51b-604c-4159-b239-2b189238a869",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "userId": "787a9868-6ec6-4174-a4fa-adadbc00c653",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "f6f7b6db-ab43-4120-aeb2-d51a8863b003",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "userId": "744ddaa8-3e79-4785-920e-12c0ee09b246",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "9eebc83e-3dbc-4f9f-8b38-b8260b877cb8",
+        "firstName": "Anonymous",
+        "sum": 5,
+        "userId": "8965b1cf-2679-4165-a3fe-39532cd2f76d",
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
+    {
+        "id": "cdb17164-bf1e-4963-a613-72e951f60292",
+        "sum": 5,
+        "ideaId": "6ae1ea20-6e79-45f8-8265-eba8a3d22677"
+    },
```

Comment:
This donation has no `userId` and no `firstName`. This doesn't look right

Reply:
Fixed